get-started/fine-tuning/api.md

```diff
@@ -0,0 +1,189 @@
+
+## Prerequisites
+
+Before getting started with fine-tuning, ensure you have the following:
+
+- **kluster.ai account**: Sign up on the [kluster.ai platform](https://platform.kluster.ai/signup){target=_blank} if you do not have one.
+- **kluster.ai API key**: After signing in, go to the [API Keys](https://platform.kluster.ai/apikeys){target=_blank} section and create a new key. For detailed instructions, see the [Get an API key](https://docs.kluster.ai/get-started/get-api-key/){target=_blank} guide.
+- **Prepared dataset**: You need data formatted according to kluster.ai's requirements for fine-tuning (detailed below).
+
+## Supported models
+
+kluster.ai currently supports fine-tuning for two base models:
+
+- [klusterai/Meta-Llama-3.1-8B-Instruct-Turbo](https://docs.kluster.ai/get-started/models/){target=_blank}
+- [klusterai/Meta-Llama-3.3-70B-Instruct-Turbo](https://docs.kluster.ai/get-started/models/){target=_blank} 
+
+!!! info
+    You can query the [models endpoint](https://docs.kluster.ai/api-reference/reference/#list-supported-models){target=_blank} in the API and filter for the tag "fine-tunable."
+
+## Fine-tuning workflow
+
+The process of fine-tuning a model using the kluster.ai API involves several key steps:
+
+
+### Prepare your data
+
+High-quality, well-formatted data is crucial for successful fine-tuning:
+
+- **Format**: Data must be in JSONL format, where each line is a valid JSON object representing a training example.
+- **Structure**: Each JSON object should contain a `messages` array with system, user, and assistant messages.
+- **Example format**:
+
+```json
+{
+  "messages": [
+    {
+      "role": "system",
+      "content": "You are a JSON Generation Specialist. Convert user requests into properly formatted JSON."
+    },
+    {
+      "role": "user",
+      "content": "Create a configuration for a web application with name 'TaskMaster', version 1.2.0, and environment set to development."
+    },
+    {
+      "role": "assistant",
+      "content": "{\n  \"application\": {\n    \"name\": \"TaskMaster\",\n    \"version\": \"1.2.0\",\n    \"environment\": \"development\"\n  }\n}"
+    }
+  ]
+}
+```
+
+- **Quantity**: The minimum requirement is 10 examples, but more diverse and high-quality examples yield better results.
+- **Quality**: Ensure your data accurately represents the task you want the model to perform.
+
+!!! tip "Data preparation"
+    For a detailed walkthrough of data preparation, see the [Fine-tuning Sentiment Analysis Tutorial](https://docs.kluster.ai/tutorials/klusterai-api/finetuning-sent-analysis/#get-the-data){target=_blank}.
+
+!!! example "Find Llama datasets on Hugging Face"
+    There is a wide range of datasets suitable for Llama model fine-tuning on [Hugging Face Datasets](https://huggingface.co/datasets?sort=trending&search=llama){target=_blank}. Browse trending and community-curated datasets to accelerate your data preparation.
+
+### Set up the client
```

Comment:
Ideally, we create a single file and then reference lines of the file. The idea is that we then at then end put the file to that people can copy paste

Reply:
Done